sql/core/src/test/scala/org/apache/spark/sql/StreamTest.scala

```diff
@@ -75,6 +76,8 @@ trait StreamTest extends QueryTest with Timeouts {
   /** How long to wait for an active stream to catch up when checking a result. */
   val streamingTimeout = 10.seconds
 
+  val outputMode: OutputMode = Append
```

Comment:
Can add this later when we have more support for more output modes and mixed suites